requirements.txt

```diff
@@ -5,41 +5,41 @@
 #    pip-compile --output-file=requirements.txt setup.py
 #
 adage==0.10.3             # via reana-commons, yadage
-alembic==1.7.6            # via flask-alembic, reana-db
-amqp==5.0.9               # via kombu
+alembic==1.7.7            # via flask-alembic, reana-db
+amqp==5.1.0               # via kombu
```

Comment:
Please don't bump all the package versions on `maint-0.8`, it's better to stay on the same version in maintenance branches.